checkly/consts.go

```diff
@@ -0,0 +1,7 @@
+package checkly
+
+import (
+	"time"
+)
+
+const apiCallTimeout = 15 * time.Second
```

Comment:
Can we also add an env for this that overrides the default value? Call me a paranoid but you never know

Reply:
@umutuzgur sure, done!